packages/client/mutations/handlers/handleUpdateSpotlightResults.ts

```diff
@@ -43,7 +43,9 @@ const handleUpdateSpotlightResults = (
     })
   }
   // ungrouping created a new group or was added to a group in the kanban
-  if ((reflectionsCount === 1 && wasInSpotlight) || (isOldGroupEmpty && !isInSpotlight)) {
```

Comment:
To test, add `isOldGroupEmpty` back and see the following:
- open Spotlight modal by clicking on a reflection called "kittens"
- add result "cats" to source "kittens" so the source group has 2 reflections
- ungroup "cats" without dropping it on another reflection group
- see that "cats" is not added to the results